logicmonitor/config.go

```diff
@@ -12,18 +12,14 @@ type Config struct {
 }
 
 //NewLMClient initialization
-func (c *Config) NewLMClient() (*lmv1.DefaultApi, error) {
-	config := lmv1.NewConfiguration()
-	config.APIKey = map[string]map[string]string{
-		"Authorization": {
-			"AccessID":  c.AccessID,
-			"AccessKey": c.AccessKey,
-		},
-	}
-	config.BasePath = "https://" + c.Company + ".logicmonitor.com/santaba/rest"
-	config.UserAgent = "LogicMonitor Terraform Provider"
+func (c *Config) newLMClient() (*lmclient.LMSdkGo, error) {
+	config := lmclient.NewConfig()
 
-	api := lmv1.NewDefaultApi()
-	api.Configuration = config
-	return api, nil
+	config.SetAccessID(&c.AccessID)
+	config.SetAccessKey(&c.AccessKey)
+	config.SetAccountDomain(&c.Company)
+	client := lmclient.New(config)
+	//	config.SetUserAgent("LogicMonitor Terraform Provider")
```

Comment:
might as well remove this line. i don't anticipate being able to set the user agent any time in the remote future

Reply:
done, will show up in next update